### DISCUSSION TITLE
My repository is not appearing among the repositories of any topic I’ve set

### DISCUSSION THREAD

**Discussion by hjk789** [community/community#18604] [Other Feature Feedback, Questions, & Ideas]
My repository (github.com/hjk789/NXEnhanced) is not appearing in the list of repositories of any topic I’ve set. My other repositories are appearing normally. This one is the only one that doesn’t. Am I missing something? Is this a bug or what?

I asked this [in the forum](https://github.community/t/my-repository-is-not-appearing-among-the-repositories-of-any-topic/256011), and it seems to be a bug.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by hjk789** [reply]
Thanks, I've reported now.

**Reply by hjk789** [reply]
They re-indexed my repository. It's fixed now.
